spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/Engine.scala

```diff
@@ -124,16 +126,13 @@ object Engine {
       "For test purpose, set bigdl.disableCheckSysEnv to true"
 
   /**
-   * Notice: Please use property DL_ENGINE_TYPE to set engineType.
-   * Default engine is MklBlas
+   * Notice: Please use property bigdl.engineType to set engineType.
+   * Default engine is mklblas
    */
   private var engineType: EngineType = {
-    val dlEngineType = System.getenv("DL_ENGINE_TYPE")
-
-    if (dlEngineType == null || dlEngineType.toLowerCase == "mklblas") {
-      MklBlas
-    } else {
-      throw new Error(s"Unknown DL_ENGINE_TYPE. $ENV_VAR_ERROR")
+    System.getProperty("bigdl.engineType", "mklblas").toLowerCase(Locale.ROOT) match {
+      case "mklblas" => MklBlas
+      case _ => throw new IllegalArgumentException(s"Unknown engine type. $ENV_VAR_ERROR")
```

Comment:
Done. Should I try to remove the local mode flag or is that best left for another time, to consider separately?